Doc/tutorial/errors.rst

```diff
@@ -18,10 +18,10 @@ Syntax errors, also known as parsing errors, are perhaps the most common kind of
 complaint you get while you are still learning Python::
 
    >>> while True print('Hello world')
-     File "<stdin>", line 1
-       while True print('Hello world')
-                      ^
-   SyntaxError: invalid syntax
+  File "<stdin>", line 1
+    while True print('Hello world')
+          ^^^^^^^^^^^^^^^^^^^^^^^^^
+SyntaxError: invalid syntax. Perhaps you forgot a comma?
 
 The parser repeats the offending line and displays a little 'arrow' pointing at
```

Comment:
This no longer matches the example.

Reply:
Hi @iritkatriel - just checking if you need anything from my side ??

Reply:
The sentence just after the example looks like it needs to be updated as well.

Reply:
I've updated the sentence as requested - let me know if it looks good to you ?

Reply:
The sentence below is now incorrect, it talks about a missing colon before `print`.

Reply:
^ This is still the case, requesting changes to make sure this is not missed.